storage-resize-images/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Version 0.1.2
+
+feature - Add new param for deleting the original image.
+fixed - Fixed "cold start" errors experienced when the Extension runs after a period of inactivity (issue #48).
```

Comment:
I don't think this was done on the `storage-resize-images` extension, since it doesn't use a firestore trigger.

Reply:
Great catch! That was meant for delete-user-data. Thanks!